engine/src/main/java/org/terasology/engine/core/module/ModuleListDownloader.java

```diff
@@ -59,13 +59,13 @@ public ModuleRegistry call() throws IOException {
                 String json = gson.toJson(jObject);
 
                 ModuleMetadata meta = metaReader.read(new StringReader(json));
-                logger.debug("Read module {} - {}", meta.getId(), meta.getVersion());
+                logger.atDebug().addArgument(() -> meta.getId()).addArgument(() -> meta.getVersion()).log("Read module {} - {}");
                 modules.add(new Module(meta, new EmptyFileSource(), Collections.emptyList(), new Reflections(),
                         (c) -> false));
             }
 
             int count = modules.size();
-            logger.info("Retrieved {} {}", count, (count == 1) ? "entry" : "entries");
+            logger.atInfo().addArgument(() -> count).addArgument(() -> (count == 1) ? "entry" : "entries").log("Retrieved {} {}");
```

Comment:
simple integer comparison so impact neglectable
```suggestion
            logger.info("Retrieved {} {}", count, (count == 1) ? "entry" : "entries"); //NOPMD
```